controllers/glance_controller.go

```diff
@@ -696,12 +701,11 @@ func (r *GlanceReconciler) generateServiceConfigMaps(
 	cms := []util.Template{
 		// ScriptsConfigMap
 		{
-			Name:               fmt.Sprintf("%s-scripts", instance.Name),
-			Namespace:          instance.Namespace,
-			Type:               util.TemplateTypeScripts,
-			InstanceType:       instance.Kind,
-			AdditionalTemplate: map[string]string{"common.sh": "/common/common.sh"},
```

Comment:
Is this related to this change?

Reply:
Yes, I'm basically removing `AdditionalTemplate: map[string]string{"common.sh": "/common/common.sh"},` as it's not used anymore.

Reply:
It seems the diff is bigger than it has to be; a whitespace issue. I don't think that's a problem though. I think the real difference here is that `AdditionalTemplate: map[string]string{"common.sh": "/common/common.sh"},` is removed.